test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Great work adding the `.gitignore` file to your project and ignoring the `test.md` file. However, following the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) it seems like you created and pushed the `test.md` file before creating the `.gitignore` file, which is why it's still visible in your repository. 
To resolve this issue, please delete the `test.md` file and push your changes again. This should ensure that the file is properly ignored and not pushed to the repository. 💪